### PR TITLE
Backport "HBASE-25861 Correct the usage of Configuration#addDeprecation (#3249)" to branch-2

### PR DESCRIPTION
**Pull request by ndimiduk**
Signed-off-by: Nick Dimiduk <ndimiduk@apache.org>

### CODE REVIEW

**Comment by ndimiduk** [comment]
FYI @ZhaoBQ , looks like your patch applies well enough to branch-2. Can you confirm that it can be applied without modification -- the configs you manipulate in this branch don't need some different action for the next 2.x release, and that there are not other uses of `addDeprecation` on branch-2 that need to be corrected? Thanks for the contribution!

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 46s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ branch-2 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 16s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m  3s |  branch-2 passed  |
| +1 :green_heart: |  compile  |  10m 53s |  branch-2 passed  |
| +1 :green_heart: |  checkstyle  |   2m 51s |  branch-2 passed  |
| +0 :ok: |  refguide  |   4m 59s |  branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  spotbugs  |  21m 52s |  branch-2 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 17s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m 13s |  the patch passed  |
| +1 :green_heart: |  compile  |  14m 27s |  the patch passed  |
| +1 :green_heart: |  javac  |  14m 27s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   3m 24s |  root: The patch generated 0 new + 58 unchanged - 3 fixed = 58 total (was 61)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  2s |  The patch has no ill-formed XML file.  |
| +0 :ok: |  refguide  |   5m 20s |  patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  hadoopcheck  |  16m 34s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1.  |
| +1 :green_heart: |  spotbugs  |  19m 40s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 54s |  The patch does not generate ASF License warnings.  |
|  |   | 124m  2s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3271 |
| Optional Tests | dupname asflicense javac spotbugs hadoopcheck hbaseanti checkstyle compile xml refguide |
| uname | Linux 6399b106467e 4.15.0-142-generic #146-Ubuntu SMP Tue Apr 13 01:11:19 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 1bb16fea5b |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
| refguide | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/artifact/yetus-general-check/output/branch-site/book.html |
| refguide | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/artifact/yetus-general-check/output/patch-site/book.html |
| Max. process+thread count | 126 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server hbase-mapreduce . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/console |
| versions | git=2.17.1 maven=3.6.3 spotbugs=4.2.2 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 31s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  7s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ branch-2 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 16s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 19s |  branch-2 passed  |
| +1 :green_heart: |  compile  |   2m 53s |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   8m 36s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   5m  1s |  branch-2 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 17s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   5m  9s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m 52s |  the patch passed  |
| -0 :warning: |  javac  |   3m 52s |  root generated 1 new + 33 unchanged - 1 fixed = 34 total (was 34)  |
| +1 :green_heart: |  shadedjars  |   9m 42s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   6m 10s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 184m 55s |  root in the patch passed.  |
|  |   | 236m 35s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3271 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux a7ea992c4b5c 4.15.0-65-generic #74-Ubuntu SMP Tue Sep 17 17:06:04 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 1bb16fea5b |
| Default Java | AdoptOpenJDK-11.0.10+9 |
| javac | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/artifact/yetus-jdk11-hadoop3-check/output/diff-compile-javac-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/testReport/ |
| Max. process+thread count | 6025 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server hbase-mapreduce . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 46s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  6s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ branch-2 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 17s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 35s |  branch-2 passed  |
| +1 :green_heart: |  compile  |   2m 52s |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   7m 59s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   4m  5s |  branch-2 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m 28s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 42s |  the patch passed  |
| -0 :warning: |  javac  |   2m 42s |  root generated 1 new + 33 unchanged - 1 fixed = 34 total (was 34)  |
| +1 :green_heart: |  shadedjars  |   7m 45s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   4m 20s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 330m 39s |  root in the patch failed.  |
|  |   | 374m  5s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/artifact/yetus-jdk8-hadoop2-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3271 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 403c1020d25c 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 1bb16fea5b |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
| javac | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/artifact/yetus-jdk8-hadoop2-check/output/diff-compile-javac-root.txt |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/artifact/yetus-jdk8-hadoop2-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/testReport/ |
| Max. process+thread count | 4762 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server hbase-mapreduce . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/1/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-common/src/main/java/org/apache/hadoop/hbase/HBaseConfiguration.java [hunk 1]
```diff
@@ -77,6 +81,39 @@ private static void checkDefaultsVersion(Configuration conf) {
...
+        "hbase.server.allocator.pool.enabled"),
+      new DeprecationDelta("hbase.ipc.server.reservoir.initial.max",
+        "hbase.server.allocator.max.buffer.count"),
+      new DeprecationDelta("hbase.ipc.server.reservoir.initial.buffer.size",
+        "hbase.server.allocator.buffer.size"),
+      new DeprecationDelta("hlog.bulk.output", "wal.bulk.output"),
+      new DeprecationDelta("hlog.input.tables", "wal.input.tables"),
+      new DeprecationDelta("hlog.input.tablesmap", "wal.input.tablesmap"),
+      new DeprecationDelta("hbase.master.mob.ttl.cleaner.period",
+        "hbase.master.mob.cleaner.period")
```
Comment (ZhaoBQ) [inline hbase-common/src/main/java/org/apache/hadoop/hbase/HBaseConfiguration.java:113]: branch-2 doesn't have this configuration, so it needs to be deleted. The rest doesn't need to change.

Comment (ndimiduk) [reply hbase-common/src/main/java/org/apache/hadoop/hbase/HBaseConfiguration.java]: Like this?

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  9s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +1 :green_heart: |  hbaseanti  |   0m  0s |  Patch does not have any anti-patterns.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ branch-2 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 16s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 35s |  branch-2 passed  |
| +1 :green_heart: |  compile  |   8m 46s |  branch-2 passed  |
| +1 :green_heart: |  checkstyle  |   2m  5s |  branch-2 passed  |
| +0 :ok: |  refguide  |   3m 50s |  branch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  spotbugs  |  11m 38s |  branch-2 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 19s |  the patch passed  |
| +1 :green_heart: |  compile  |   8m 42s |  the patch passed  |
| +1 :green_heart: |  javac  |   8m 42s |  the patch passed  |
| +1 :green_heart: |  checkstyle  |   2m  4s |  root: The patch generated 0 new + 58 unchanged - 3 fixed = 58 total (was 61)  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  1s |  The patch has no ill-formed XML file.  |
| +0 :ok: |  refguide  |   3m 20s |  patch has no errors when building the reference guide. See footer for rendered docs, which you should manually inspect.  |
| +1 :green_heart: |  hadoopcheck  |  11m 42s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1.  |
| +1 :green_heart: |  spotbugs  |  12m 25s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 54s |  The patch does not generate ASF License warnings.  |
|  |   |  82m 39s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3271 |
| Optional Tests | dupname asflicense javac spotbugs hadoopcheck hbaseanti checkstyle compile xml refguide |
| uname | Linux 0c0699a94e8a 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 1a8d3c3b6b |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
| refguide | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/2/artifact/yetus-general-check/output/branch-site/book.html |
| refguide | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/2/artifact/yetus-general-check/output/patch-site/book.html |
| Max. process+thread count | 141 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server hbase-mapreduce . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/2/console |
| versions | git=2.17.1 maven=3.6.3 spotbugs=4.2.2 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   2m 43s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  7s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ branch-2 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 46s |  branch-2 passed  |
| +1 :green_heart: |  compile  |   3m  9s |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   7m 33s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   4m 29s |  branch-2 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 12s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m 31s |  the patch passed  |
| +1 :green_heart: |  compile  |   3m  7s |  the patch passed  |
| -0 :warning: |  javac  |   3m  7s |  root generated 1 new + 33 unchanged - 1 fixed = 34 total (was 34)  |
| +1 :green_heart: |  shadedjars  |   7m 33s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   4m 24s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 240m 28s |  root in the patch passed.  |
|  |   | 285m 53s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/2/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3271 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 033a767c345b 4.15.0-136-generic #140-Ubuntu SMP Thu Jan 28 05:20:47 UTC 2021 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 1a8d3c3b6b |
| Default Java | AdoptOpenJDK-11.0.10+9 |
| javac | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/2/artifact/yetus-jdk11-hadoop3-check/output/diff-compile-javac-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/2/testReport/ |
| Max. process+thread count | 4074 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server hbase-mapreduce . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/2/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   4m  0s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  7s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ branch-2 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 16s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 33s |  branch-2 passed  |
| +1 :green_heart: |  compile  |   2m  8s |  branch-2 passed  |
| +1 :green_heart: |  shadedjars  |   5m 59s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m 29s |  branch-2 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 15s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 12s |  the patch passed  |
| -0 :warning: |  javac  |   2m 12s |  root generated 1 new + 33 unchanged - 1 fixed = 34 total (was 34)  |
| +1 :green_heart: |  shadedjars  |   5m 57s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m 33s |  the patch passed  |
||| _ Other Tests _ |
| +1 :green_heart: |  unit  | 321m 17s |  root in the patch passed.  |
|  |   | 359m  4s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/2/artifact/yetus-jdk8-hadoop2-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/3271 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 477ccf1ae865 4.15.0-58-generic #64-Ubuntu SMP Tue Aug 6 11:12:41 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | branch-2 / 1a8d3c3b6b |
| Default Java | AdoptOpenJDK-1.8.0_282-b08 |
| javac | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/2/artifact/yetus-jdk8-hadoop2-check/output/diff-compile-javac-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/2/testReport/ |
| Max. process+thread count | 4662 (vs. ulimit of 12500) |
| modules | C: hbase-common hbase-server hbase-mapreduce . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-3271/2/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: hbase-common/src/main/java/org/apache/hadoop/hbase/HBaseConfiguration.java (hunk 1 above)
Comment (ZhaoBQ) [reply hbase-common/src/main/java/org/apache/hadoop/hbase/HBaseConfiguration.java]: Yes, there is nothing else to modify.

**Pull request merged**